owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ *   ownCloud Android client application
+ *
+ *   @author David González Verdugo
+ *   Copyright (C) 2019 ownCloud GmbH.
+ *
+ *   This program is free software: you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License version 2,
+ *   as published by the Free Software Foundation.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.owncloud.android.ui.activity
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.owncloud.android.R
+import info.hannes.logcat.BothLogsFragment
+import info.hannes.logcat.LogcatFragment
+import info.hannes.timber.fileLoggingTree
+import timber.log.Timber
+
+class LogHistoryActivity : ToolbarActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContentView(R.layout.logs)
+        setupToolbar()
+
+        // Check that the activity is using the layout version with the fragment_container FrameLayout
+        if (findViewById<View>(R.id.fragment_container) != null) {
+
+            // However, if we're being restored from a previous state,
+            // then we don't need to do anything and should return or else
+            // we could end up with overlapping fragments.
+            if (savedInstanceState != null) {
+                return
+            }
+
+            var logFragment: Fragment? = null
+
+            Timber.forest().fileLoggingTree()?.let {
+                logFragment = BothLogsFragment.newInstance(
+                    "Owncloud.log",
+                    "search logfile",
+                    "search logcat",
+                    getString(R.string.mail_logger)
+                )
+            } ?: run {
+                logFragment = LogcatFragment.newInstance(
+                    "logfile.log",
+                    "search logcat",
+                    getString(R.string.mail_logger)
```

Comment:
Same here

Reply:
It's the previous logic. 
Only he knows, what's the purpose
<img width="622" alt="image" src="https://user-images.githubusercontent.com/3314607/68924668-9f1ee880-0781-11ea-9c75-06612b7486ae.png">

Which default email address should it have ? Or simply remove it ?

Reply:
I don't know which is its purpose, i meant moving hardcoded strings to constants.

By the way, it would be good if you push branches within oC repository. This will help us when we need to perform a rebase, and we won't disturb you each time 👍

Reply:
A: hardcoded strings
It's a developer option ! Do you want translate _logcat_  ?

B: enable continue on oC side
Here it is https://github.com/owncloud/android/pull/2725